app/controllers/sprints_controller.rb

```diff
@@ -59,6 +59,19 @@ def create
         raise 'Fail to update project with product backlog' unless @project.save!
       end
       flash[:notice] = l(:notice_successful_create)
+      pbi_params = {"issue"=>{"tracker_id"=> Tracker.find_by_name('Product Backlog').id, 
+                              "sprint_id"=> @sprint.id, 
+                              "subject"=>"#{@sprint.name} pbi", 
+                              "status_id"=> IssueStatus.find_by_name('In Progress').id , 
+                              "priority_id"=> Enumeration.find_by_name('Normal').id, 
+                              "assigned_to_id"=>"", "start_date"=>@sprint.sprint_start_date, 
+                              "due_date"=>@sprint.sprint_end_date, 
+                              "estimated_hours"=>"", "done_ratio"=>"0", 
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.